src/grpc_server.cc

```diff
@@ -237,9 +241,28 @@ namespace grpc_labview
         if (_server != nullptr)
         {
             // We need shutdown passing a deadline so that any RPC calls in progress are terminated as well.
-            _server->Shutdown(std::chrono::system_clock::now());
-            _server->Wait();
-            _runThread->join();
+            _server->Shutdown();
```

Comment:
Some questions:
a) why did we remove the deadline for server->shutdown() call here?
b) aren't we suppose to wait for server->shutdown() to complete

Reply:
You are right. I should not modify this.

Reply:
The code is updated.